runtime/src/main/java/org/corfudb/runtime/view/SequencerView.java

```diff
@@ -1,24 +1,35 @@
 package org.corfudb.runtime.view;
 
+import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.AccessLevel;
 import org.corfudb.protocols.wireprotocol.TokenResponse;
 import org.corfudb.protocols.wireprotocol.TxResolutionInfo;
 import org.corfudb.runtime.CorfuRuntime;
-import org.corfudb.runtime.clients.SequencerClient;
 import org.corfudb.util.CFUtils;
 
-import java.util.Map;
 import java.util.Set;
 import java.util.UUID;
 
+
 /**
  * Created by mwei on 12/10/15.
  */
+
 public class SequencerView extends AbstractView {
 
+    // TODO the failover sequencer should be a separate class as specified
+    // in https://github.com/CorfuDB/CorfuDB/wiki/Quorum-Replication
```

Comment:
Sequencer failover is now allowed only within a full-fledged layout-change. It establishes a new layout, as well as a new epoch. Therefore, there is no need to manage sequencer leases as previously described in the wiki.

A recovery-sequencer is not yet implemented, but that's a different story. It is on my TODO list, and is relatively simple.